bin/hubot

```diff
@@ -1,168 +1,9 @@
 #!/usr/bin/env coffee
-# vim:ft=coffee ts=2 sw=2 et :
-# -*- mode:coffee -*-
 
-Hubot    = require '..'
+# While all other files have been converted to JavaScript via https://github.com/github/hubot/pull/1347,
+# we left the `bin/hubot` file to remain in CoffeeScript in order prevent
+# breaking existing 3rd party adapters of which some are still written in
+# CoffeeScript themselves. We will depracate and eventually remove this file
+# in a future version of hubot
```

Comment:
I don’t think `require('coffee-script')` will be enough, I'll have to investigate it. If you don’t mind I’d make a follow up PR to change it later if it works.

Regarding the deprecation warning I’m not yet sure when we drop CoffeeScript support officially, but we can add a deprecation warning in a patch release, too, so we don’t keep that PR waiting